code/modules/mob/mob.dm

```diff
@@ -207,6 +207,8 @@
 			else
 				msg = alt_msg
 				type = alt_type
+				if(type & MSG_AUDIBLE && !can_hear())
+					return
 
 		if(type & MSG_AUDIBLE && !can_hear())//Hearing related
```

Comment:
You early-return when this is the case earlier. Is this intentional?

Reply:
nope just realized it would skip over the second if all together, good catch